demo/kserve/scripts/install/2.required-crs.sh

```diff
@@ -0,0 +1,99 @@
+#!/bin/bash
+# Environment variables
+# - CHECK_UWM: Set this to "false", if you want to skip the User Workload Configmap check message
+# - TARGET_OPERATOR: Set this among odh, rhods or brew, if you want to skip the question in the script.
+set -o pipefail
+set -o nounset
+set -o errtrace
+# set -x   #Uncomment this to debug script.
+
+source "$(dirname "$(realpath "$0")")/../env.sh"
+source "$(dirname "$(realpath "$0")")/../utils.sh"
+
+echo
+info "Let's create required CRs and required setup"
+
+if [[ ! -d ${BASE_DIR} ]]
+then
+  mkdir ${BASE_DIR}
+fi
+
+if [[ ! -d ${BASE_CERT_DIR} ]]
+then
+  mkdir ${BASE_CERT_DIR}
+fi
+
+# Create an istio instance
+echo
+light_info "[INFO] Create an istio instance"
+echo
+oc create ns istio-system -oyaml --dry-run=client | oc apply -f-
+oc::wait::object::availability "oc get project istio-system" 2 60
+
+oc apply -f custom-manifests/service-mesh/smcp.yaml
+wait_for_pods_ready "app=istiod" "istio-system"
+wait_for_pods_ready "app=istio-ingressgateway" "istio-system"
+wait_for_pods_ready "app=istio-egressgateway" "istio-system"
+
+oc wait --for=condition=ready pod -l app=istiod -n istio-system --timeout=300s
+oc wait --for=condition=ready pod -l app=istio-ingressgateway -n istio-system --timeout=300s
+oc wait --for=condition=ready pod -l app=istio-egressgateway -n istio-system --timeout=300s
+
+# kserve/knative
+echo
+light_info "[INFO] Update SMMR"
+echo
+oc create ns knative-serving -oyaml --dry-run=client | oc apply -f-
+oc::wait::object::availability "oc get project knative-serving" 2 60
+
+oc apply -f custom-manifests/service-mesh/default-smmr.yaml
+
+# Create a Knative Serving installation
+echo
+light_info "[INFO] Create a Knative Serving installation"
+echo
+oc apply -f custom-manifests/serverless/knativeserving-istio.yaml
+
+wait_for_pods_ready "app=controller" "knative-serving"
+wait_for_pods_ready "app=net-istio-controller" "knative-serving"
+wait_for_pods_ready "app=net-istio-webhook" "knative-serving"
+wait_for_pods_ready "app=autoscaler-hpa" "knative-serving"
+wait_for_pods_ready "app=domain-mapping" "knative-serving"
+wait_for_pods_ready "app=webhook" "knative-serving"
+oc delete pod -n knative-serving -l app=activator --force --grace-period=0
+oc delete pod -n knative-serving -l app=autoscaler --force --grace-period=0
+wait_for_pods_ready "app=activator" "knative-serving"
+wait_for_pods_ready "app=autoscaler" "knative-serving"
+
+oc wait --for=condition=ready pod -l app=controller -n knative-serving --timeout=300s
```

Comment:
```suggestion
oc wait --for=condition="ready pod" -l app=controller -n knative-serving --timeout=300s
```

and on the others.

Reply:
I don't get your suggestion. It is not right syntax for `--for`

Reply:
do you mean that you can't use double quotes on `--for`?